.github/workflows/build.yml

```diff
@@ -48,3 +48,8 @@ jobs:
         env:
           DOCKER_USER: ${{ secrets.DOCKER_USER }}
           DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
+      - if: startsWith(github.ref, 'refs/heads/master')
+        run: make docker-manifest-annotate-unstable docker-manifest-push-unstable
```

Comment:
Go releaser must run before this docker push thingy, otherwise there are no images to push. 

I see unstable as latest [while not being named latest](https://vsupalov.com/docker-latest-tag/). I think I rather do regular releases instead of updating unstable on every push.

Reply:
Oh, I see, there is a dependency on go releaser...

It doesn't have to be though, if you want we can also explore automating this whole thing? To use `echoip` as an example, I think we would need to add some things to `Dockerfile` to let it self-build the binary ([something like this](https://github.com/mpolden/echoip/blob/994c2529e9c81204eda469bcb0ddad9f9bb9eb30/Dockerfile#L1-L7)), and then build all the architectures for example [like this](https://github.com/mpolden/echoip/blob/994c2529e9c81204eda469bcb0ddad9f9bb9eb30/Makefile#L42-L48).

Something to consider, in general it's very nice to be able to just grab the image from the current master to test something, before reporting an issue or while waiting for the next release 🙂 But if you want to keep it as it is, lets just close the PR 👍

Reply:
>  if you want we can also explore automating this whole thing? 

Well it is already automated via goreleaser (:, I mean, we wouldn't to build the binaries multiple times for the same arch (one time in docker and one time via goreleaser). Yes, we can build it in docker and then copy the binaries out for uploading them to the github release, but this would be the same as just running goreleaser on every master push.

I think if screego would be a project which had a rapid development with not frequent releases, then I would agree with you, but currently there is often a release after the master push, so I don't see much benefit in screego for this.